sdk/core/core-http/src/fetchHttpClient.ts

```diff
@@ -153,14 +154,20 @@ export abstract class FetchHttpClient implements HttpClient {
       const response: CommonResponse = await this.fetch(httpRequest.url, requestInit);
 
       const headers = parseHeaders(response.headers);
+
+      const isBodyReallyStream = isResponseBodyStream(
+        httpRequest.operationSpec,
+        response.status.toString()
+      );
+
       const operationResponse: HttpOperationResponse = {
         headers: headers,
         request: httpRequest,
         status: response.status,
-        readableStreamBody: httpRequest.streamResponseBody
```

Comment:
We probably also want to remove all usage of `httpRequest.streamResponseBody`? It's in public surface but I don't see other usages than checking for response body type and doing it incorrectly.

Reply:
Certainly we should remove it from `core-https` at least.

Reply:
Looking at it again. It is used in browser xhrHttpClient to have different code paths to resolve the responses so I will keep it.